src/constants/krs-providers.js

```diff
@@ -2,5 +2,9 @@ export default [
   {
     label: 'Keyternal',
     value: 'keyternal'
+  },
+  {
+    label: 'BitGoKRS',
```

Comment:
```suggestion
    label: 'BitGo KRS',
```

Could we do a space here?